spec/nest_thermostat_spec.rb

```diff
@@ -17,10 +21,9 @@ module NestThermostat
     end
 
     it "does not remember the login email or password" do
-      nest = Nest.new(email: ENV['NEST_EMAIL'], password: ENV['NEST_PASS'], temperature_scale: :fahrenheit)
-
-      expect(nest).not_to respond_to(:email)
-      expect(nest).not_to respond_to(:password)
+      @nest = Nest.new(email: 'test@yahoo.com', password: 'sekret', temperature_scale: :fahrenheit)
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.